pkg/rm/remoting_parser_api.go

```diff
@@ -0,0 +1,116 @@
+package rm
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/pkg/errors"
+	"github.com/seata/seata-go/pkg/common/log"
+)
+
+const (
+	RemotingUnknow  = 0
+	RemotingGrpc    = 1
+	RemotingDubbogo = 2
+	RemotingLocal   = 3
+)
+
+type RemotingParser interface {
+	IsService(target interface{}) bool
+	IsReference(target interface{}) bool
+	IsRemoting(target interface{}) bool
+	ParseService(target interface{}) (*TwoPhaseAction, error)
+	ParseReference(target interface{}) (*TwoPhaseAction, error)
+	ParseTwoPhaseActionByInterface(target interface{}) (*TwoPhaseAction, error)
+	GetRemotingType(target interface{}) int
+}
+
+var (
```

Comment:
only one var, remove the (  , just `var remotingParseTable = make([]RemotingParser, 0)`

Reply:
thx, have done.